.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @powerhome/heroes-for-hire
\ No newline at end of file
+* @powerhome/heroes-for-hire @garettarrowood
```

Comment:
I'm fine with the authors part, which is about credit, but I don't think this is necessary. If you want to stay up to date, you can just watch all activity on the repo. I'd like to avoid individual owners where possible, and I'm not sure what the benefit is here.

Reply:
```suggestion
* @powerhome/heroes-for-hire
```

Reply:
Sounds good